src/main/java/coms/w4156/moviewishlist/Controllers/MovieController.java

```diff
@@ -0,0 +1,69 @@
+package coms.w4156.moviewishlist.controllers;
```

Comment:
This file is pretty close copy of the WishlistController with some code cleanup.